studio/components/layouts/AccountLayout/AccountLayout.tsx

```diff
@@ -91,7 +89,7 @@ const AccountLayout = ({ children, title, breadcrumbs }: PropsWithChildren<Accou
               },
               // show the MFA page only if the feature flag is set or the user has already MFA setup.
               // He should be able to edit/revoke his MFA even if MFA feature flag is disabled.
-              ...(mfaSetup || (factors?.all || []).length > 0
+              ...((factors?.all || []).length > 0
```

Comment:
Remove the conditional and the comment.

Reply:
Don't we still only want to show these if the user has setup MFA?

Reply:
ah nvm